public/app/features/plugins/plugin_loader.ts

```diff
@@ -166,21 +169,47 @@ export function importDataSourcePlugin(meta: DataSourcePluginMeta): Promise<Gene
   });
 }
 
-export function importAppPlugin(meta: PluginMeta): Promise<AppPlugin> {
-  const isAngular = meta.angular?.detected ?? meta.angularDetected;
-  const fallbackLoadingStrategy = meta.loadingStrategy ?? PluginLoadingStrategy.fetch;
-  return importPluginModule({
+export class DisabledAppPlugin extends Error {}
+export class NotAnAppPlugin extends Error {}
```

Comment:
Are these two being used?

Reply:
No, good point 👍 Removed in https://github.com/grafana/grafana/pull/86624/commits/4b4e36152b3ab43a1edee88703dd8ab050637de9.